### PR TITLE
Add Infura Sepolia Faucet to docs #3906

### PR DESCRIPTION
**Pull request by anataliocs**
- [ ] Because this PR includes a **bug fix**, relevant tests have been included.
- [ ] Because this PR includes a **new feature**, the change was previously discussed on an Issue or with someone from the team.
- [X] Docs update

---

**ISSUE:**  https://github.com/NomicFoundation/hardhat/issues/3906

**Description:**  Docs update.

Adding url to Infura faucet, https://www.infura.io/faucet/sepolia to docs.

**Article:**

https://hardhat.org/tutorial/deploying-to-a-live-network

### CODE REVIEW

File: docs/src/content/tutorial/deploying-to-a-live-network.md [hunk 1]
```diff
@@ -121,7 +121,8 @@ To deploy on Sepolia you need to send some Sepolia ether to the address that's g
 
 - [Alchemy Sepolia Faucet](https://sepoliafaucet.com/)
 - [Coinbase Sepolia Faucet](https://coinbase.com/faucets/ethereum-sepolia-faucet) (only works if you are using the Coinbase Wallet)
-
+- [Infura Sepolia Faucet](https://www.infura.io/faucet/sepolia)
+-
```
Comment (anataliocs) [inline docs/src/content/tutorial/deploying-to-a-live-network.md:125]:
```suggestion

```

**Comment by anataliocs** [comment]
@fvictorio Quick 1 line update to docs :)

**Comment by anataliocs** [comment]
@erin-at-work

**Comment by fvictorio** [comment]
Thanks!

**Pull request merged**